user-manual/administer/settings.rst

```diff
@@ -2215,6 +2216,104 @@ To save any modifications, click the "Save" button located below the
 
 :ref:`Back to top <settings>`
 
+
+.. _storage-service:
+
+Storage Service
+===============
+In this section, :term:`administrators <administrator>` can enable the
+download of original files and Archival Information Packages (AIPs) from an 
+`Archivematica <https://archivematica.org>`_ digital preservation system that 
+has uploaded Dissemination Information Package (DIP) access copies to AtoM.
+
+.. IMPORTANT::
+
+   Both the **arStorageServicePlugin** and **arRestApiPlugin** must be enabled 
+   for this "Storage service" option to appear on the Settings menu and for 
+   the Archivematica integration to work. See :ref:`Manage Plugins <
```

Comment:
@peterVG here's the error. Sometimes Sphinx craps out when a line break interrupts a ref link. I see the opening bracket is on the line above (2233). I'd suggest just moving everything in the last sentence with the link to its own line. E.g.

```
.. IMPORTANT::

   Both the **arStorageServicePlugin** and **arRestApiPlugin** must be enabled 
   for this "Storage service" option to appear on the Settings menu and for 
   the Archivematica integration to work. See:
   :ref:`Manage plugins <manage-plugins>`. 
```

Reply:
done